index.js

```diff
@@ -30,6 +30,36 @@ async function bcc() {
   return
 }
 
+async function xgboost() {
+  let artifacts_version = core.getInput('artifacts_version');
+  core.debug(artifacts_version);
+  if (artifacts_version === undefined || artifacts_version == null || artifacts_version.length === 0) {
```

Comment:
Can we convert this to a function? Something like:
```
const DEFAULT_ARTIFACTS_VERSION="0.26.0";
const getInputOrDefault = (input, defaultValue) => {
   if (input === undefined || input === null){
      return defaultValue;
   }
   return input;
};
...
let artifacts_version = getInputOrDefault(core.getInput('artifacts_version'),DEFAULT_ARTIFACTS_VERSION);
```

Reply:
done